// _@vuepress_core@1.5.2@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "/Users/zhangqin/Desktop/blog/node_modules/_@vuepress_core@1.5.2@@vuepress/core/lib/client/components/GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-40d6e052",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-40d6e052").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-77e90d60",
    path: "/article/%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84%E5%92%8C%E7%AE%97%E6%B3%95/JavaScript%E7%BC%96%E7%A0%81%E8%83%BD%E5%8A%9B.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-77e90d60").then(next)
    },
  },
  {
    path: "/article/数据结构和算法/JavaScript编码能力.html",
    redirect: "/article/%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84%E5%92%8C%E7%AE%97%E6%B3%95/JavaScript%E7%BC%96%E7%A0%81%E8%83%BD%E5%8A%9B.html"
  },
  {
    name: "v-2b3d3189",
    path: "/article/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-2b3d3189").then(next)
    },
  },
  {
    path: "/article/index.html",
    redirect: "/article/"
  },
  {
    name: "v-735772ad",
    path: "/article/%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84%E5%92%8C%E7%AE%97%E6%B3%95/%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-735772ad").then(next)
    },
  },
  {
    path: "/article/数据结构和算法/数据结构.html",
    redirect: "/article/%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84%E5%92%8C%E7%AE%97%E6%B3%95/%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84.html"
  },
  {
    name: "v-4a3df9c9",
    path: "/article/%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84%E5%92%8C%E7%AE%97%E6%B3%95/%E7%AE%97%E6%B3%95.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4a3df9c9").then(next)
    },
  },
  {
    path: "/article/数据结构和算法/算法.html",
    redirect: "/article/%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84%E5%92%8C%E7%AE%97%E6%B3%95/%E7%AE%97%E6%B3%95.html"
  },
  {
    name: "v-30096ca4",
    path: "/article/JavaScript%E5%9F%BA%E7%A1%80/%E8%AF%AD%E6%B3%95%E5%92%8CAPI.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-30096ca4").then(next)
    },
  },
  {
    path: "/article/JavaScript基础/语法和API.html",
    redirect: "/article/JavaScript%E5%9F%BA%E7%A1%80/%E8%AF%AD%E6%B3%95%E5%92%8CAPI.html"
  },
  {
    name: "v-60c49ead",
    path: "/article/JavaScript%E5%9F%BA%E7%A1%80/%E5%8E%9F%E5%9E%8B%E5%92%8C%E5%8E%9F%E5%9E%8B%E9%93%BE.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-60c49ead").then(next)
    },
  },
  {
    path: "/article/JavaScript基础/原型和原型链.html",
    redirect: "/article/JavaScript%E5%9F%BA%E7%A1%80/%E5%8E%9F%E5%9E%8B%E5%92%8C%E5%8E%9F%E5%9E%8B%E9%93%BE.html"
  },
  {
    name: "v-73cae38a",
    path: "/article/%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84%E5%92%8C%E7%AE%97%E6%B3%95/%E6%89%8B%E5%8A%A8%E5%AE%9E%E7%8E%B0%E5%89%8D%E7%AB%AF%E8%BD%AE%E5%AD%90.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-73cae38a").then(next)
    },
  },
  {
    path: "/article/数据结构和算法/手动实现前端轮子.html",
    redirect: "/article/%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84%E5%92%8C%E7%AE%97%E6%B3%95/%E6%89%8B%E5%8A%A8%E5%AE%9E%E7%8E%B0%E5%89%8D%E7%AB%AF%E8%BD%AE%E5%AD%90.html"
  },
  {
    name: "v-5e846cf1",
    path: "/article/JavaScript%E5%9F%BA%E7%A1%80/%E6%89%A7%E8%A1%8C%E6%9C%BA%E5%88%B6.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5e846cf1").then(next)
    },
  },
  {
    path: "/article/JavaScript基础/执行机制.html",
    redirect: "/article/JavaScript%E5%9F%BA%E7%A1%80/%E6%89%A7%E8%A1%8C%E6%9C%BA%E5%88%B6.html"
  },
  {
    name: "v-037c620f",
    path: "/article/JavaScript%E5%9F%BA%E7%A1%80/%E5%8F%98%E9%87%8F%E5%92%8C%E7%B1%BB%E5%9E%8B.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-037c620f").then(next)
    },
  },
  {
    path: "/article/JavaScript基础/变量和类型.html",
    redirect: "/article/JavaScript%E5%9F%BA%E7%A1%80/%E5%8F%98%E9%87%8F%E5%92%8C%E7%B1%BB%E5%9E%8B.html"
  },
  {
    name: "v-4523d015",
    path: "/article/JavaScript%E5%9F%BA%E7%A1%80/%E4%BD%9C%E7%94%A8%E5%9F%9F%E5%92%8C%E9%97%AD%E5%8C%85.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4523d015").then(next)
    },
  },
  {
    path: "/article/JavaScript基础/作用域和闭包.html",
    redirect: "/article/JavaScript%E5%9F%BA%E7%A1%80/%E4%BD%9C%E7%94%A8%E5%9F%9F%E5%92%8C%E9%97%AD%E5%8C%85.html"
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
